### ISSUE TITLE
Se queda en blanco

### ISSUE THREAD

**Issue by Wishengan** [Cuevana/storm#165]
Cuando abro directamente me abre el programa pero todo en blanco como si no llegase a cargar nada. Tengo w7 ultimate sp1, saque el antivirus y lo agregue a la lista del firewall.

**Comment by LhalOo** [comment]
solo tienes que reiniciar el internet. 
Apaga y prende tu rauter para q cambie el ip

**Comment by Wishengan** [comment]
No, no tengo router, igual reinicie el modem pero ese no es el problema. Ya hace rato que no puedo usar storm q siempre tengo el mismo problema, lo voy viendo cada tanto a ver si cambia algo pero nada.

**Comment by LhalOo** [comment]
Mmm. Bueno a mi lo q me pasaba era q abria el programa y noe cargaba nada. Lo que Asia era pagar el router o si no le cambiaba el IP por cmd. Luego abria y normal.

**Comment by fredygtrs** [comment]
tengo el mismo problema ahora, me abre el programa pero no carga nada, ni la lista de peliculas sale.
tu lo solucionaste? que puedo hacer?

**Comment by Wishengan** [comment]
Al final nadie me ayudo y no hice nada especial, espere unos cuantos dias hasta que pusieron otra version y me anduvo, pero no es que lo logre arreglar con algo :(
